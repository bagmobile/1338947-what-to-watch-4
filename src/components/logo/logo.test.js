import React from "react";
import renderer from "react-test-renderer";
import Logo from "./logo";
import {BrowserRouter, Route} from "react-router-dom";

describe(`Logo component`, () => {

  it(`Render logo default`, () => {
    const tree = renderer
      .create(
          <BrowserRouter>
            <Route>
              <Logo/>
            </Route>
          </BrowserRouter>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Render logo with light`, () => {
    const tree = renderer
      .create(<BrowserRouter>
        <Route>
          <Logo isLight={true}/>
        </Route>
      </BrowserRouter>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

});
