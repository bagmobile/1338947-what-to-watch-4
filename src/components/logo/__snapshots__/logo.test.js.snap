// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component Render logo default 1`] = `
<div
  className="logo"
>
  <a
    className="logo__link"
    href="/"
    onClick={[Function]}
  >
    <span
      className="logo__letter logo__letter--1"
    >
      W
    </span>
    <span
      className="logo__letter logo__letter--2"
    >
      T
    </span>
    <span
      className="logo__letter logo__letter--3"
    >
      W
    </span>
  </a>
</div>
`;

exports[`Logo component Render logo with light 1`] = `
<div
  className="logo"
>
  <a
    className="logo__link logo__link--light"
    href="/"
    onClick={[Function]}
  >
    <span
      className="logo__letter logo__letter--1"
    >
      W
    </span>
    <span
      className="logo__letter logo__letter--2"
    >
      T
    </span>
    <span
      className="logo__letter logo__letter--3"
    >
      W
    </span>
  </a>
</div>
`;
